src/components/Stepper/Steps.tsx

```diff
@@ -6,11 +6,19 @@ import { getStep } from '../../const/steps';
 import { MailTemplateCtx } from '../../contexts/mail-template.context';
 import { SpreadsheetCtx } from '../../contexts/spreadsheet.context';
 import { StepCtx } from '../../contexts/step.context';
+import { useStyles } from '../../hooks/useStyles';
 import { Step } from '../../models';
 import { mailContent } from '../../seeds/mail';
 import Navigation from './Navigation';
 
+const styles = {
+  mainContent: {
+    paddingBottom: 56,
```

Comment:
This means padding bottom would be `56px`. This value seems random. If you really want to use spacing, use `theme.spacing.unit` or think of `flex` display

Reply:
It is up to you guys... I used flex, you overwrite my changes on master, and the final result was that send button was hover by navigation